audiofile/core/utils.py

```diff
@@ -62,18 +62,40 @@ def run(shell_command):
 def run_ffmpeg(infile, outfile, offset, duration):
     """Convert audio file to WAV file."""
     if duration:
-        cmd = f'ffmpeg -ss {offset} -i "{infile}" -t {duration} "{outfile}"'
+        cmd = [
+                  'ffmpeg',
+                  '-ss', str(offset),
+                  '-i', infile,
+                  '-t', str(duration),
+                  outfile
+              ]
```

Comment:
Adjusting to the other styles here as well:

```suggestion
        cmd = [
              'ffmpeg',
              '-ss', str(offset),
              '-i', infile,
              '-t', str(duration),
              outfile
          ]
```